packages/views/screens/managers/hwWalletLogin/selectAccount/accountCard.js

```diff
@@ -5,9 +5,9 @@ import { truncateAddress } from '@wallet/utilities/account';
 import { TertiaryButton } from '@basics/buttons';
 import WalletVisual from '@wallet/detail/identity/walletVisual';
 import LiskAmount from '@shared/liskAmount';
-import { Input } from '@basics/inputs';
-import Icon from '@basics/icon';
-import Tooltip from '@basics/tooltip/tooltip';
+import { Input } from 'src/theme';
+import Icon from 'src/theme/Icon';
```

Comment:
I think Icon can be added as part of the named import in theme/index.js so that it can be called together with Input like this

`import { Input, Icon } from 'src/theme';`

Reply:
It actually will take more time to build